coding_style/README.adoc

```diff
@@ -72,20 +114,20 @@ Multiple conditions that all need to be true (a logical `and`) can also be speci
 [source,yaml]
 ----
 tasks:
-  - name: Print a message
-    ansible.builtin.debug:
-      msg: This is how it's done.
+- name: Print a message
```

Comment:
this formatting contradicts our own rules: [Indent list contents beyond the list definition](https://github.com/redhat-cop/automation-good-practices/tree/main/coding_style#yaml-and-jinja2-syntax) the same applies to l. 125 ff.

Reply:
Agreed and fixed.